docs/sources/docker-hub/accounts.md

```diff
@@ -34,21 +34,52 @@ page.
 
 ## Organizations and groups
 
-Also available on the Docker Hub are organizations and groups that allow
-you to collaborate across your organization or team. You can see what
-organizations [you belong to and add new organizations](
+To help you collaborate across your projects or teams, Docker Hub also
+provides organizations and groups that allow more than one Hub user to work on
+shares repositories.
+
+### Creating and viewing organizations
+
+You can see what organizations [you belong to and add new organizations](
 https://hub.docker.com/account/organizations/) from the Account Settings
-tab. They are also listed below your user name on your repositories page and in your account profile.
+tab. They are also listed below your user name on your repositories page
+and in your account profile.
 
 ![organizations](/docker-hub/hub-images/orgs.png)
 
-From within your organizations you can create groups that allow you to
-further manage who can interact with your repositories.
+### Organization groups
+
+Users that are in the `Owners` group for an organization, can create and
```

Comment:
No comma needed after "organization". Simplify to "Users in the `Owners` group..."